### ISSUE TITLE
Slow down in Game even with speedup options enabled

### ISSUE THREAD

**Issue by mobileandroidpsp** [hrydgard/ppsspp#17459]
### Game or games this happens in

ULUS10365 - Lego Indiana Jones: The Original Adventures

### What area of the game

![Screenshot_20230514-144257](https://github.com/hrydgard/ppsspp/assets/133598682/82ab09b7-9ae9-4cdb-8c8d-a6e2c14dbea8)
The majority of the cutscene and rest of gameplay is around 40-77% on average.

### Speed seen in PPSSPP

44%

### Platform

Android

### Mobile phone model or graphics card

Schok Volt SV55 v30 (armeabi-v7a)

### PPSSPP version affected

V1.15.3

### Graphics backend (3D API)

OpenGL / GLES

### Any other notes or things you've tried

Trying many combinations of speedhacks and options enabled, resetting to default. Restarting device and uninstalling apps for speedup.

### Checklist

- [X] Test in the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already fixed.
- [X] [Search for other reports](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same issue.
- [X] Try resetting settings or older versions and include if the issue is related.
- [X] Try changing graphics settings to determine if one causes the slowdown.
- [X] Include logs or screenshots of issue.

**Comment by Panderner** [comment]
The problem is your phone is not powerful enough to run this game at full speed.

**Comment by ANR2ME** [comment]
Yup, the specs are rather low, and as i remembered Lego Indiana Jones 2 is pretty heavy on some scenes on my phone (not sure about the prequel tho, but might be the same)
![image](https://github.com/hrydgard/ppsspp/assets/7974720/ec2b89b2-bf68-44b5-9d2d-0391f4cce264)

**Comment by Panderner** [comment]
> (not sure about the prequel tho, but might be the same)

LEGO Indiana Jones 2 runs at 30 FPS cap while Indiana Jones 1 runs at unlocked 60 FPS.

**Comment by mobileandroidpsp** [comment]
> Yup, the specs are rather low, and as i remembered Lego Indiana Jones 2 is pretty heavy on some scenes on my phone (not sure about the prequel tho, but might be the same) ![image](https://user-images.githubusercontent.com/7974720/238293296-ec2b89b2-bf68-44b5-9d2d-0391f4cce264.png)

Was able to use ppsspp v1.0 and get much better speeds, 85-100% speed. Yes the hardware is not powerful enough but using an older apk, results were improved because hardware like the gpu and cpu are from 2014-2016 era even if the device itself was released in 2021-2022 with android 11 go. Ram is 2gb.

**Comment by mobileandroidpsp** [comment]
> The problem is your phone is not powerful enough to run this game at full speed.

Yes, 30fps games run just fine. Its the 60fps games that give trouble because of the hardware limitations.

**Comment by chip-64bit** [comment]
> I see this as a performance regression that I want to resolve eventually. Going back to early versions is not a solution.

Hi, this issue is also present in 1.18.1.
